pkg/storkctl/backuplocation.go

```diff
@@ -147,75 +147,67 @@ func newGetBackupLocationCommand(cmdFactory Factory, ioStreams genericclioptions
 	return getBackupLocationCommand
 }
 
-func s3BackupLocationPrinter(backupLocationList *storkv1.BackupLocationList, writer io.Writer, options printers.PrintOptions) error {
+func s3BackupLocationPrinter(
+	backupLocationList *storkv1.BackupLocationList,
+	options printers.GenerateOptions,
+) ([]metav1beta1.TableRow, error) {
 	if backupLocationList == nil {
-		return nil
+		return nil, nil
 	}
+
+	rows := make([]metav1beta1.TableRow, 0)
 	for _, backupLocation := range backupLocationList.Items {
-		name := printers.FormatResourceName(options.Kind, backupLocation.Name, options.WithKind)
-		if options.WithNamespace {
-			if _, err := fmt.Fprintf(writer, "%v\t", backupLocation.Namespace); err != nil {
-				return err
-			}
-		}
-
-		if _, err := fmt.Fprintf(writer, "%v\t%v\t%v\t%v\t%v\t%v\t%v\n",
-			name,
-			backupLocation.Location.Path,
-			backupLocation.Location.S3Config.AccessKeyID,
-			backupLocation.Location.S3Config.SecretAccessKey,
-			backupLocation.Location.S3Config.Region,
-			backupLocation.Location.S3Config.Endpoint,
-			backupLocation.Location.S3Config.DisableSSL,
-		); err != nil {
-			return err
-		}
+		row := getRow(&backupLocation,
+			[]interface{}{backupLocation.Name,
+				backupLocation.Location.Path,
+				backupLocation.Location.S3Config.AccessKeyID,
+				backupLocation.Location.S3Config.SecretAccessKey,
```

Comment:
we should not print secrets in storkctl output right?

Reply:
They are changed to `<HIDDEN>` by default in `newGetBackupLocationCommand()`

Reply:
They are changed to `<HIDDEN>` by default in `newGetBackupLocationCommand()`